test_expect_failure/jmh/BUILD

```diff
@@ -0,0 +1,12 @@
+load(
+    "//jmh:jmh.bzl",
+    "scala_benchmark_jmh",
+)
+
+scala_benchmark_jmh(
+    name = "jmh_reports_failure",
+    srcs = [
+        "InvalidBenchmark.scala",
+        "ValidBenchmark.scala",
```

Comment:
Is this mandatory for the test?

Reply:
I initially thought so. Without the fix, the `bazel build` step will pass but the `bazel run` step will fail (because you'll end up running zero benchmarks, which causes an error). We're testing with `bazel build` though, so it's probably not necessary.

Reply:
Removed it and verified the test still works as expected. 🙂